### PR TITLE
Exchange updates

### PR DESCRIPTION
**Pull request by kaprests**
Visual updates and addition of some field as requested by PostKom.

### CODE REVIEW

File: nablapps/exchange/views.py [hunk 1]
```diff
@@ -6,23 +6,26 @@
...
-    queryset = ExchangeNewsArticle.objects.order_by("-pk")
+    context_object_name = "ex_list"
+
+    def get_queryset(self):
+        query = self.request.GET.get("q")
+        queryset = University.objects.order_by("land")
+        if query:
+            queryset = queryset.filter(
+                Q(land__icontains=query) | Q(univ_navn__icontains=query)
+            )
```
Comment (Amund211) [inline nablapps/exchange/views.py:21]: v cool

**Review by Amund211 — APPROVED** [review]
Looks good to me! Just one comment about some trailing whitespace.

File: nablapps/exchange/templates/exchange/exchange_frontpage.html [hunk 2]
```diff
@@ -1,70 +1,55 @@
-{% extends "base.html" %}
-{% load easy_thumbnails %}
-{% load sekizai_tags %}
+{% extends "base.html" %}    
+{% load easy_thumbnails %}    
+{% load sekizai_tags %}
```
[the remark this answers is not in the document]
Comment (kaprests) [reply nablapps/exchange/templates/exchange/exchange_frontpage.html:3]: Nope. That's just a fluke :)

[the remark this answers is not in the document]
Comment (kaprests) [reply nablapps/exchange/templates/exchange/exchange_frontpage.html:3]: Fluke is apparently a positive accident, so a negative version of a fluke I guess :stuck_out_tongue:

File: nablapps/exchange/views.py (hunk 1 above)
Comment (kaprests) [reply nablapps/exchange/views.py:21]: I cannot take credit for those lines however, they were part of the old view for the university list which was not authored by me. I only slightly restructured the views and moved the lines around :upside_down_face:

**Pull request merged**